soh/soh/Enhancements/game-interactor/GameInteractor_Hooks.h

```diff
@@ -69,6 +69,9 @@ void GameInteractor_RegisterOnAssetAltChange(void (*fn)(void));
 //Mark: - Pause Menu
 void GameInteractor_ExecuteOnKaleidoUpdate();
 
+//Mark - Randomizer options spawning EnItem00 actors
+void EnItem00_DrawRandomizedItem(EnItem00* enItem00, PlayState* play);
```

Comment:
This function was already used by other things than just pots though. I understand it's different from the other GameInteractor_Hooks functions, but externing it on the pot side feels dirtier than just having it in that header to me.

Reply:
I disagree with this being dirty for `ShufflePots.cpp`, it is pretty standard to forward declare things in our files (like functions from the game source, etc) (you were already doing something similar with externing `Randomizer_GetSettingValue` in ShufflePots too).

The function was only used in `hook_handlers.cpp`, and now additionally for `ShufflePots.cpp`.

I don't want us to stick this in a random header for the sake of having it "be in a header", especially since it is not related to GameInteractor in the first place. That is dirtier imo. Having it in just `ShufflePots.cpp` clarifies who is actually dependent on it, so when we decompose `hooks_handler.cpp` we can approach a solution that is better.

Reply:
It might not be related to GameInteractor but in my mind hook_handlers.cpp is a direct extension of said hooks. But whatever, I'll change it but I'm honestly not happy about it. Maybe that comes from my dislike of hook_handlers.cpp in general, I don't know.